src/main/java/com/minecolonies/rankup/Rankup.java

```diff
@@ -66,13 +70,17 @@
     private final ConfigurationLoader<CommentedConfigurationNode> loader;
     private final SubInjectorModule subInjectorModule = new SubInjectorModule();
 
-    public EconomyService econ;
+    public PermissionsUtils perms;
+    public ConfigUtils      configUtils;
+    public AccountingUtils  accUtils;
+    public EconomyService   econ;
     public static Rankup instance = null;
     public        Game                                         game;
     private final RankupCommand                                rankupCommand;
     private final Path                                         configDir;
     private       GuiceObjectMapperFactory                     factory;
     private       Map<Class<? extends BaseConfig>, BaseConfig> configs;
+    private       List<GroupsConfig>                           groupConfigs;
     private       Injector                                     RankupInjector;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "RankupInjector" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS00116)